effekt/shared/src/main/scala/effekt/generator/llvm/Transformer.scala

```diff
@@ -417,8 +417,9 @@ object Transformer {
   def transform(tpe: machine.Type): Type = tpe match {
     case machine.Positive()          => positiveType
     case machine.Negative()          => negativeType
-    case machine.Type.Int()          => NamedType("Int")
-    case machine.Type.Double()       => NamedType("Double")
+    case machine.Type.Int()          => IntegerType64()
+    case machine.Type.Byte()         => IntegerType8()
+    case machine.Type.Double()       => DoubleType()
```

Comment:
@phischu Is this ok, or should I revert this?

Reply:
No, this is fine.